adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.m

```diff
@@ -88,26 +88,35 @@ - (void)renderNativeAdForAdConfiguration:
                  strongSelf->_nativeAd = nativeAd;
                  strongSelf->_nativeAd.delegate = strongSelf;
                  strongSelf->_nativeAd.rootViewController = adConfiguration.topViewController;
-
-                 if (strongSelf->_loadCompletionHandler) {
-                   id<GADMediationNativeAdEventDelegate> delegate =
-                       strongSelf->_loadCompletionHandler(strongSelf, nil);
-                   strongSelf->_delegate = delegate;
-                 }
+      
+                 [strongSelf loadRequireData];
                }];
 }
 
-#pragma mark - GADMediationNativeAd
-
-- (GADNativeAdImage *)icon {
-  if (!_icon) {
-    if (_nativeAd.data.icon && _nativeAd.data.icon.imageURL != nil) {
-      _icon = [self imageWithUrlString:_nativeAd.data.icon.imageURL];
-    }
-  }
-  return _icon;
+- (void)loadRequireData {
+    NSString *urlString = _nativeAd.data.icon.imageURL;
```

Comment:
Is it guaranteed that we can get the `imageURL` successfully every time through accessing the property?

Reply:
Do we need the check below?
```
if (_nativeAd.data.icon && _nativeAd.data.icon.imageURL != nil)
```

Reply:
I changed the code.

Reply:
`- loadRequireData` is invoked when PAGLNativeAd loadAd callback success.So _nativeAd.data isn't nil.